test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -568,14 +568,21 @@ protected void disablePreparedReuseForTest()
 
     protected String createType(String query)
     {
-        String typeName = "type_" + seqNumber.getAndIncrement();
+        String typeName = createTypeName();
```

Comment:
`createTypeName` already adds the name to `types`, so it doesn't need to be added in this method, at line 573.

Reply:
This also happens on the PR for 3.11